src/components/UserListItem/UserListItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Avatar, IconButton, ListItem, ListItemAvatar, ListItemText } from "@mui/material";
+import { observer } from "mobx-react";
+import { Person, Delete } from "@mui/icons-material";
+
+interface Props {
+    id: number;
+    firstName: string;
+    lastName: string;
+    age: number;
+}
+
+const UserListItem = observer(({id, firstName, lastName, age}: Props) => {
```

Comment:
Naming - we use multiple Users in List. And you already have UsersList in code belowe